test/scripts/integration/test-integration.py

```
@@ -0,0 +1,1348 @@
+        # GET /olla/models/{id} using discovered model
+        if self.selected_model:
+            r = self._get(f"/olla/models/{self.selected_model}")
+            ok = False
+            detail = ""
+            if r and r.status_code == 200:
+                ok = True
+                detail = f"found {self.selected_model}"
+            else:
+                detail = self._error_detail(r)
+
+            self._print_result(f"Model lookup (/olla/models/{{id}})", ok, detail)
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)